### DISCUSSION TITLE
Are unnumbered sections supported?

### DISCUSSION THREAD

**Discussion by osenan** [davidgohel/officedown#89] [Q&A] [answered]
Hi! The following code does not produce unnumbered section:
```
---
title: "Habits"
author: John Doe
date: March 22, 2005
output: officedown::rdocx_document
---

# Hello World! {.unnumbered}
```

Changing `{.unnumbered}` for `{-}` does not work either. Is it possible to have unnumbered sections that escape from the TOC?

Thank you!

**Comment by alapo** [comment, marked as answer]
I ran able into a similar issue a few weeks ago. The method does seem to make it so it is not in the TOC. I was able to find a weird way to get it to not be numbered. I have a walkthrough in the video below if anyone is interested. The template file is attached here, but I show how to do it in the video. 

[![IMAGE ALT TEXT](http://img.youtube.com/vi/4C3akvTrK3Q/0.jpg)](http://www.youtube.com/watch?v=4C3akvTrK3Q "Unnumbered Heading using OfficeDown")

[manuscript-template-unum.docx](https://github.com/davidgohel/officedown/files/9635613/manuscript-template-unum.docx)
